packages/docusaurus/src/webpack/utils.ts

```diff
@@ -477,37 +484,7 @@ function getTerserParallel() {
 }
 
 export function getMinimizer(useSimpleCssMinifier = false): Plugin[] {
-  const minimizer = [
-    new TerserPlugin({
-      cache: true,
-      parallel: getTerserParallel(),
-      sourceMap: false,
-      terserOptions: {
-        parse: {
-          // we want uglify-js to parse ecma 8 code. However, we don't want it
-          // to apply any minification steps that turns valid ecma 5 code
-          // into invalid ecma 5 code. This is why the 'compress' and 'output'
-          // sections only apply transformations that are ecma 5 safe
-          // https://github.com/facebook/create-react-app/pull/4234
-          ecma: 8,
-        },
-        compress: {
-          ecma: 5,
-          warnings: false,
-        },
-        mangle: {
-          safari10: true,
-        },
-        output: {
-          ecma: 5,
-          comments: false,
-          // Turned on because emoji and regex is not minified properly using default
-          // https://github.com/facebook/create-react-app/issues/2488
-          ascii_only: true,
-        },
-      },
-    }),
-  ];
+  const minimizer = [new ESBuildMinifyPlugin({target: 'es2017'})];
```

Comment:
Switched output to es2017, since v2 doesn't plan to support ie11, and all other major browsers have good support for es2017: https://caniuse.com/?search=es2017